docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -26,8 +26,8 @@ The main features of the Image Util API include:
   You can [decode images](#decode), [encode them](#encode), and [encode animation](#animation) with the following formats:
 
   - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
-  - Input image formats for decoding: JPEG, PNG, GIF, BMP, and WEBP
-  - Output image formats for encoding: JPEG, PNG, GIF, BMP, and WEBP
+  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP, HEIF, and JPEG-XL
+  - Output image formats for encoding: JPEG, PNG, GIF, BMP, WEBP, and JPEG-XL
```

Comment:
Global
@jiyong2,
Please let me know if "**JPEG-XL**" be written as "**JPEG XL**"?
During a search on the internet, I found it to be written as "**JPEG XL**"
Ref: https://jpeg.org/jpegxl/

Reply:
@omar-srbd 
Oh, it's my mistake. I thought that it was a compound noun.
You are right, "JPEG XL" is a original.
I will replace the word to "JPEG XL". Thank you~!

Reply:
I updated it.